src/handlers/message.ts

```
@@ -50,7 +50,7 @@ class ChatActionManager {
 
   constructor(
     private ctx: Context,
-    private chatId: number
+    private chatId: number,
```

ESLint

[error] Delete `,`

(prettier/prettier)